src/table/TableBodyWrapper.jsx

```diff
@@ -39,7 +39,7 @@ const TableBodyWrapper = ({ rootElement, tableData, constraints, selectionsAPI,
     const selectionsEnabled = !!selectionsAPI && !constraints.active;
     selDispatch({ type: 'set-enabled', payload: { isEnabled: selectionsEnabled } });
     setColumnRenderers(getColumnRenderers(selectionsEnabled));
-  }, [constraints]);
+  }, [constraints, columns.length]);
```

Comment:
invoking setColumnRenderers when measure or dimension is updated, which means when the length of columns is changed, columnRenderers should be updated